src/types.ts

```diff
@@ -198,3 +200,12 @@ export class ApiError extends Error {
     this.data = response.data
   }
 }
+
+type TextContentType = {
+  content:
+    | { 'text/plain': any }
+    | { 'text/css': any }
+    | { 'text/csv': any }
+    | { 'text/html': any }
```

Comment:
I'd add `text/xml` as a common option
```suggestion
    | { 'text/html': any }
    | { 'text/xml': any }
```